detail/harvesterhci.io.host/index.vue

```diff
@@ -41,14 +41,27 @@ export default {
 
     const hash = {
       nodes:         this.$store.dispatch('harvester/findAll', { type: NODE }),
-      hostNetworks:  this.$store.dispatch('harvester/findAll', { type: HCI.NODE_NETWORK }),
-      longhornNodes: this.$store.dispatch(`${ inStore }/findAll`, { type: LONGHORN.NODES }),
-      blockDevices:  this.$store.dispatch(`${ inStore }/findAll`, { type: HCI.BLOCK_DEVICE }),
       pods:          this.$store.dispatch(`${ inStore }/findAll`, { type: POD }),
     };
 
+    if (this.$store.getters['harvester/schemaFor'](HCI.NODE_NETWORK)) {
+      hash.hostNetworks = this.$store.dispatch('harvester/findAll', { type: HCI.NODE_NETWORK });
+    } else {
+      this.hasHostNetworksSchema = false;
+    }
```

Comment:
Can we change hasHostNetworksSchema as a computed? 
```suggestion
    if (this.hasHostNetworksSchema) {
      hash.hostNetworks = this.$store.dispatch('harvester/findAll', { type: HCI.NODE_NETWORK });
    }
    
    // computed
    hasHostNetworksSchema() {
      return this.$store.getters['harvester/schemaFor'](HCI.NODE_NETWORK)
    }
```